homeassistant/components/temper/sensor.py

```diff
@@ -55,7 +56,9 @@ def setup_platform(
     for idx, dev in enumerate(temper_devices):
         if idx != 0:
             name = f"{prefix}_{idx!s}"
-        TEMPER_SENSORS.append(TemperSensor(dev, name, scaling))
+        TEMPER_SENSORS.append(TemperatureSensor(dev, name, scaling))
+        if dev.lookup_humidity_offset(0) is not None:
+            TEMPER_SENSORS.append(HumiditySensor(dev, name))
```

Comment:
Seems like this integration could benefit from using EntityDescriptions instead.

Reply:
:+1: I've update these to use `SensorEntityDescription`